recipes/libgpg-error/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class GPGErrorConan(ConanFile):
+    name = "libgpg-error"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gnupg.org/software/libgpg-error/index.html"
+    topics = ("conan", "gpg", "gnupg")
+    description = "Libgpg-error is a small library that originally defined common error values for all GnuPG " \
+                  "components."
+    license = "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+
+    exports_sources = "patches/**"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux is supported")
```

Comment:
Can you elaborate? Is it the library that only supports Linux, or is it the recipe that only works on Linux?
It would be nice to make this clearer in the error message.

Reply:
@madebr, what should I write if I don't want to develop this recipe to support all operating systems, now Linux will do. If one wants to build this package for Windows than there will be another PR in the future

Reply:
If it is because of the recipe not being prepared for Macos/Windows, then it is better to clarify it, something like:

```suggestion
            raise ConanInvalidConfiguration("This recipe only support Linux. You can contribute Windows and/or Macos support.")
```